src/System.ComponentModel/tests/CancelEventArgsTests.cs

```diff
@@ -38,4 +38,4 @@ public void Cancel_Set_GetReturnsExpected()
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
for future references, don't or limit making changes outside of what you are trying to address. This change isn't related to your PR so it shouldn't be in here. You can of course leave it here now as it is just formatting.